fontir/src/variations.rs

```diff
@@ -50,12 +50,12 @@ impl RoundTiesEven for f64 {
     }
 }
 
-impl RoundTiesEven for f32 {
-    #[inline]
-    fn round_ties_even(self) -> f32 {
-        self.round_ties_even()
-    }
-}
+//impl RoundTiesEven for f64 {
```

Comment:
Was this always unnecessary? Can it just be removed?

Reply:
oops good catch! this used to say `f32` before I /s/f32/f64, and then became redundant.